hs_core/views/resource_folder_hierarchy.py

```diff
@@ -225,6 +226,24 @@ def data_store_folder_unzip(request):
         content_type="application/json"
     )
 
+@api_view(['POST'])
+def data_store_folder_unzip_public(request, pk, pathname):
+    """
+    Public version of data_store_folder_unzip, incorporating path variables
+
+    :param request:
+    :param pk:
+    :param pathname:
+    :return HttpResponse:
+    """
+    import sys
+    sys.path.append('/pycharm-debug')
+    import pydevd
+    pydevd.settrace('10.20.1.51', port=21000, suspend=False)

```

Comment:
This is only relevant to testing in your local environment and should not be included as part of the final PR.

```python
    import sys
    sys.path.append('/pycharm-debug')
    import pydevd
    pydevd.settrace('10.20.1.51', port=21000, suspend=False)
```